### ISSUE TITLE
Requires sbt in $PATH

### ISSUE THREAD

**Issue by what-the-functor** [caiorss/clip.jar#2]
In the README it states that Java is the only dependency, however the project additionally requires SBT to build.  You might consider the inclusion of sbt-launch.jar in this repo so the user does not have to install SBT. 

P.S. Interesting use of Scala!  I found this project via /r/emacs.

**Comment by caiorss** [comment]
> In the README it states that Java is the only dependency, however the project additionally requires SBT to build. 

Yes. Java is the only dependency necessary to make the compiled code run since it is bundled with Scala rutime libraries. 

> however the project additionally requires SBT to build

Yes. As it is as Scala application it needs SBT launch and the Scala run-time itself installed. Sbt-launch alone won't build the application. The sbt config file is configured to create an Uber jar bundling Scala runtime with the App so that the user doesn't need to install the Scala runtime.

It is better to put some instructions or hyperlinks to download the Scala runtime and Sbt tool. 

> P.S. Interesting use of Scala! I found this project via /r/emacs

Scala was the only way to tame the Java API and avoid lots of not funny null pointer exceptions, null checking and case switch statements. I love the Java API, but I hate the Java language and syntax. 

I will update the building instructions.

**Comment by what-the-functor** [comment]
I'd expect a Scala developer to have Scala installed (and SBT if he/she uses it); otherwise I'd only expect a recent version of Java.  Have a look at [my branch](https://github.com/tonylotts/clip.jar/tree/simplify-build-dependencies) wherein the user does not need to have SBT or Scala installed.

If SBT is installed, make will default to the installed version of SBT, otherwise make will use the bundled sbt-launch.jar.

**Comment by caiorss** [comment]
Ok. Thanks. It really simplifies the building process. A pull request would be appreciated.

Best regards.
